npm_and_yarn/lib/dependabot/npm_and_yarn/requirement.rb

```diff
@@ -17,6 +18,7 @@ class Requirement < Gem::Requirement
       PATTERN = /\A#{PATTERN_RAW}\z/.freeze
 
       def self.parse(obj)
+        return ["=", nil] if obj.is_a?(String) && obj.strip == LATEST_REQUIREMENT
```

Comment:
This is a bit obscure but seems to work and not blow up, we can't actually compare `latest` to anything

Reply:
`satisfied_by?` always returns false